hat/assets/js/apps/Iaso/domains/orgUnits/components/OrgUnitInfos.tsx

```diff
@@ -27,6 +27,12 @@ import { OrgUnitMultiReferenceInstances } from './OrgUnitMultiReferenceInstances
 import { useGetOrgUnit } from './TreeView/requests';
 
 const useStyles = makeStyles(theme => ({
+    '@global': {
```

Comment:
I guess this syntax comes from [`jss-plugin-global`](https://cssinjs.org/jss-plugin-global/?v=v10.10.0)?

As far as I understand, this rule will be applied to `body` element on on pages containing this component. Is this correct?

Reply:
Yes indeed. Main body is limited to the screen size on the whole app to allow overflow on other components. In this specific case the dropdown is going out of the body then not visible. Applying the change on all the page leads to double scroll issues. So the workaround here is too modify the body style only if this component is visible. I'm not really a fan of it, but it's the faster solution.